civic_api_client/variants_lister.py

```diff
@@ -377,12 +377,24 @@ def get_variant_details(self, variant_id):
         variant_url = utils.civic_api_url() + 'variants/' + str(variant_id)
         return requests.get(variant_url, verify = False).json()
 
+    def prior_check(self,variant_detail):
+        """Prior checks before parsing variant details:
+        1. Fliter variant with no accepted evidence item"""
```

Comment:
Should be fixed now. 
Special thanks and apologies to @gatoravi  and @malachig  for spending so much time on my terrible spelling every day lol.